### DISCUSSION TITLE
Examples of super easy use case: Get the Address and Pass it to HTML pure HTML/JS

### DISCUSSION THREAD

**Discussion by scorpionmit01** [WalletConnect/walletconnect-monorepo#3247] [Developer Support]
I'm probably missing something, but I feel like a pretty basic use case is missing in the documentation.

1. Include a Script Tag to load the SDK
2. I click a button to Trigger the Modal Sign In
3. On Success, place the Address in a Div with id=address
4. On failure, pop an alert

Obviously, real implementation would handle those with more useful UI than that. However, I am struggling to figure out how to implement a basic "hello world" app in a truly vanilla HTML/JS environment. That's vanilla HTML/JS.

Does anyone have a trivial example they can share? I did manage to get everything compiled at one point, but doing something with it was harder.

**Comment by glitch-txs** [comment]
Hi have you check this repo?
https://github.com/WalletConnect/web3modal-examples

How to handle the sessions will depend on the SDK you choose for the integration. For web3modal you can use wagmi's actions. For ethereum-provider you can use any library like ethers.js, web3.js, viem, etc. Or the usual EIP-1193 methods

**Reply by scorpionmit01** [reply]
That repo is fantastic, thank you. This gets things much closer to straight-forward, thank you.

Here are my favorite examples:
walletconnect-modal-sign-html-cdn
walletconnect-modal-ethereum-provider-html

Is a pretty straight-forward example. It uses ES6 which is not common for vanilla HTML/JS though. But this gets me much closer, thank you.
